packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/Topics.tsx

```diff
@@ -1,7 +1,74 @@
-import React from 'react';
+import { useCommonNavigate } from 'client/scripts/navigation/helpers';
+import { CWText } from 'client/scripts/views/components/component_kit/cw_text';
+import {
+  CWTab,
+  CWTabsRow,
+} from 'client/scripts/views/components/component_kit/new_designs/CWTabs';
+import React, { useState } from 'react';
+import CreateCommunityHint from '../../CreateCommunity/components/CreateCommunityHint';
+import CreateTopicSection from './CreateTopicsSection';
+import ManageTopicsSection from './ManageTopicsSection';
+import './Topics.scss';
 
-const Topics = () => {
-  return <div className="Topics">Topics</div>;
+export const Topics = () => {
+  const TABS = [
+    { value: 'create-topic', label: 'Create Topic' },
+    { value: 'manage-topics', label: 'Manage Topics' },
+  ];
+
+  const navigate = useCommonNavigate();
+  const [selectedTab, setSelectedTab] = useState(TABS[0].value);
+
+  const updateActiveTab = (activeTab: string) => {
+    const params = new URLSearchParams();
+    params.set('tab', activeTab);
+    navigate(`${window.location.pathname}?${params.toString()}`, {}, null);
+    setSelectedTab(activeTab);
+  };
+
+  return (
+    <div className="TopicsPage">
+      <main>
+        <header>
+          <CWText type="h2">Topics</CWText>
+          <CWText type="b1" className="subheader">
+            Create topics and sub-topics, and rearrange them in the sidebar
+          </CWText>
+        </header>
+
+        <CWTabsRow>
+          {TABS.map((tab, index) => (
+            <CWTab
+              key={index}
+              label={tab.label}
+              onClick={() => updateActiveTab(tab.value)}
+              isSelected={selectedTab === tab.value}
+            />
+          ))}
+        </CWTabsRow>
+        {selectedTab === TABS[0].value ? (
+          <CreateTopicSection />
+        ) : (
+          <ManageTopicsSection />
+        )}
+      </main>
+      <aside>
+        {selectedTab === TABS[0].value ? (
+          <CreateCommunityHint
+            title="Topics and Subtopics"
+            hint="Top level topics can act as parents to subtopics. Subtopics can not have additional subtopics."
+          />
+        ) : (
+          <CreateCommunityHint
+            title="Topic Sorting"
+            hint="Drag the topics on the left to the order you want them to appear
+            on the side panel navigation of your community page. Tap the pencil
+            icon to edit the topic or delete the topic."
+          />
+        )}
```

Comment:
it would be good if on admin page we build the layout with FeatureHint the same way. Please take a look on @mzparacha PR https://github.com/hicommonwealth/commonwealth/pull/6364/files and you guys should agree on the pattern. I like @mzparacha approach because it is pixel perfect with the design and works well on mobile as well. Here it is not working on mobile 
![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/3517f7ae-d594-4a52-b1f2-3a50ec8aeeb6)

Reply:
Done!